lms/djangoapps/verify_student/api.py

```diff
@@ -33,3 +40,76 @@ def send_approval_email(attempt):
     else:
         email_context = {'user': attempt.user, 'expiration_datetime': expiration_datetime.strftime("%m/%d/%Y")}
         send_verification_approved_email(context=email_context)
+
+
+def create_verification_attempt(user, name, status, expiration_datetime=None):
```

Comment:
Nit (optional): A number of APIs use type annotations these days, which can be helpful for documentation and correctness checking.

Reply:
I've added annotations to these functions!